docs/docs/concepts/package-arch.md

```diff
@@ -0,0 +1,24 @@
+---
+id: package-arch
+title: Packages
+sidebar_label: Packages
+keywords:
+  - InstantHIE
+  - Packages
+  - Architecture
+description: A description of the instant OpenHIE package architecture
+---
+
+The fundamental concept of Instant OpenHIE is that it can be extended to support additional use cases and workflows. This is achieved through packages. A [**core package**](packages/core.md) has been produced which other packages will all derive from. A package will either extend directly from the core package or from another existing package.
+
+Each package will contain the following sorts of technical artefacts:
+* Docker compose scripts for setting up the applications required for this package’s use cases and workflows
+* Kubernetes scripts for setting up the applications required for this package’s use cases and workflows
+* Configuration scripts to setup required configuration metadata
+* Extensions to the test harness to test the added use cases with test data
+
+The below diagram shows how packages will extend off each other to add use cases of increasing complexity.
+
+![](package-arch.png)
```

Comment:
Cool, I've made the change. The one issue I have with doing it this way is that it now doesn't render well if you navigate to the markdown page on Github and in vscode when previewing where as it will if done using normal markdown format. It's not a huge deal though, seems it's the recommended approach.

Reply:
Yeah it doesnt make the preview of these docs look very nice as it doesnt render the images. I also dont like that part as its harder to see the layout. But maybe this should ideally be viewed in the context of the website and not individually.

The main reason for using the context is to ensure the path always points to the images regardless of where its deployed. Currently we have the `/instant/` path set as its being served under openhie github, but if we had a sub-domain instead the baseUrl would just be `/` and any hardcoded references would be broken